cachi2/core/models/input.py

```diff
@@ -47,7 +47,7 @@ def show_error(error: "ErrorDict") -> str:
 
 
 # Supported package managers
-PackageManagerType = Literal["gomod", "npm", "pip"]
+PackageManagerType = Literal["gomod", "npm", "pip", "yarn"]
```

Comment:
See my earlier comment, this hunk should be dropped, we already had it in the list prior to this PR.